examples/TestDriver/ios/TestDriver.xcodeproj/project.pbxproj

```diff
@@ -358,7 +358,7 @@
 			inputFileListPaths = (
 			);
 			inputPaths = (
-				"${SRCROOT}/Pods/Target Support Files/Pods-TestDriver/Pods-TestDriver-resources.sh",
+				"${PODS_ROOT}/Target Support Files/Pods-TestDriver/Pods-TestDriver-resources.sh",
```

Comment:
Looks like this is a side effect of using Cocoapods 1.6.1 instead of 1.5.3.  The change is entirely innocuous, but happy to pull it out of this PR if there any objections.